src/Ombi.Store/Entities/RequestType.cs

```diff
@@ -4,6 +4,7 @@ public enum RequestType
     {
         TvShow = 0,
         Movie = 1,
-        Album = 2,
+        Artist = 2,
```

Comment:
This might break existing requests  They will be stored in the database as type 2. But albums are now type 3